### DISCUSSION TITLE
Issue with Postal SMTP Server status

### DISCUSSION THREAD

**Discussion by kayastha1988** [postalserver/postal#2152] [Help with using Postal]
When I was checking SMTP Server status online at smtper.net I receive as in below image. Also I tried sending testing email from postal dashboard, and its coming to my inbox. Sending testing email was working indeed. But still I am not sure what's happening with my postal smtp server. I have created both SMTP credentials and API credentials for testing purpose.

How can this problem be solve?

Looking forward to hearing from you. 

Thanks
 
![image](https://user-images.githubusercontent.com/19544006/189871994-b5e7091c-1f97-4b46-9312-47c2f281502e.png)

**Comment by willpower232** [comment]
You will need to watch the SMTP server docker output to see what happens when that website tries to send an email as apparently the website isn't going to tell you.

**Reply by kayastha1988** [reply]
Managed to check SMTP server. While I was trying to send testing email via cmd line, I have got something like as shown in image below. Also in SMTP server, everything looks fine. I don't know what I am missing. Still there is something missing.
Looking forward your help.
Thanks
![image](https://user-images.githubusercontent.com/19544006/190592335-f1472dc0-244c-45e2-9c25-104ed8c7b6e3.png)

**Reply by willpower232** [reply]
I would presume then that you have not set up marketlayer.com on the credential you're using.

**Reply by kayastha1988** [reply]
No, I haven't set up that. To login postal dashboard, I have used admin@mrktl.com. I dont have this email actually. I just used this random email for testing purpose. Is this the mistake that I have made on this?

**Reply by kayastha1988** [reply]
I have tried using admin@mrktl.com and test@mrtkl.com but still I am getting same thing. Where I can find out this information in postal?

![image](https://user-images.githubusercontent.com/19544006/190604184-c2e22c02-1020-4de9-b3fd-21fd7c0bfdcd.png)

**Reply by willpower232** [reply]
you can only send messages from domains you've added in the UI (and those domains have to be available to the credential you're using)

**Reply by kayastha1988** [reply]
My mail server domain is : mail.mrktl.com and this is the only domain I have added. I couldn't get it what actually I need to in this case. 

SMTP server details:
====================
SMTP Server Address: mail.mrktl.com
Port : 25
Username: marketlayer-limited/mrktl-mailserver
Password : ********************

I guess I am almost close to SMTP server but something is still missing or I am doing wrong.
Please guide me, what I need to do.

**Reply by willpower232** [reply]
You need to go to this page in the UI and add in the domains you want to send from 

![image](https://user-images.githubusercontent.com/1619102/190667542-4f679cec-fce4-47d0-b41b-5e3be26aecf7.png)

**Reply by kayastha1988** [reply]
If this is the section, than I have already added my domain name. 
I have also got SPF, DKIM and Return Path with green check signal. 

Do I also need to define in tracking domains as well?

**Reply by willpower232** [reply]
No, tracking domains should be unrelated. If your domain name listed in that page is mrktl.com then you need to make sure you're using a credential from the tab next to it. If it isn't mrktl.com then it should be :sweat_smile:
